src/clearcase.ts

```diff
@@ -292,16 +292,25 @@ export class ClearCase {
   public async findCheckouts(): Promise<string[]> {
     let lscoArgTmpl = this.configHandler.configuration.FindCheckoutsCommand.Value;
     let results: string[] = [];
+	let resNew: string[] = [];
+	let wsf: string[] = workspace.workspaceFolders[0].uri.fsPath;
     try {
       let cmdOpts = lscoArgTmpl.split(' ');
-      await this.runCleartoolCommand(["lsco"].concat(cmdOpts), workspace.workspaceFolders[0].uri.fsPath, (data: string[]) => {
+      await this.runCleartoolCommand(["lsco"].concat(cmdOpts), wsf, (data: string[]) => {
         results = results.concat(data);
+      });
+	  results.forEach((e) => {
+	    if (e.startsWith("\\")) {
```

Comment:
I think this section is strictly windows-only. I would prefer to wrap this code in a function ( e.g. normalizePath or something ) so that we can provide an alternative implementation if needed.

Reply:
Do you think you'll need an alternative implementation for unix systems? I was thinking that the whole path alteration will only be done on windows, as unix paths won't have drive letters (=> the cleartool will always reference to workspace root or system root, which will work in both cases).
Therefore I checked for backslash to make sure, only windows paths will be altered.

Reply:
Ok, that makes sense.